src/main/kotlin/no/nav/klage/kaka/api/view/RawDataExport.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.klage.kaka.api.view
+
+import java.util.*
+
+data class AnonymizedVurdering(
+    //should be fine with saksdataId b/c we validate access when accessing saksdata
+    val id: UUID,
+    val saksdataCreated: Date,
+    val saksdataModified: Date,
+    val tilknyttetEnhet: String,
+    val hjemmelIdList: List<String>,
+    val avsluttetAvSaksbehandler: Date,
+    val ytelseId: String,
+    val utfallId: String,
+    val sakstypeId: String,
+    val mottattVedtaksinstans: Date,
+    val vedtaksinstansEnhet: String,
+    val mottattKlageinstans: Date,
+
+    val kvalitetsvurderingCreated: Date,
+    val kvalitetsvurderingModified: Date,
+    val arbeidsrettetBrukeroppfoelging: Boolean,
+    val begrunnelseForHvorforAvslagOpprettholdes: Boolean,
+    val begrunnelsenErIkkeKonkretOgIndividuell: Boolean,
+    val betydeligAvvik: Boolean,
+    val brukIOpplaering: Boolean,
+    val detErFeilIKonkretRettsanvendelse: Boolean,
+    val detErIkkeBruktRiktigHjemmel: Boolean,
+    val innholdetIRettsregleneErIkkeTilstrekkeligBeskrevet: Boolean,
+    val klagerensRelevanteAnfoerslerIkkeKommentert: Boolean,
+    val konklusjonen: Boolean,
+    val nyeOpplysningerMottatt: Boolean,
+    val oversendelsesbrevetsInnholdIkkeISamsvarMedTema: Boolean,
+    val oversittetKlagefristIkkeKommentert: Boolean,
+    val raadgivendeLegeErBruktFeilSpoersmaal: Boolean,
+    val raadgivendeLegeErBruktMangelfullDokumentasjon: Boolean,
+    val raadgivendeLegeErIkkeBrukt: Boolean,
+    val raadgivendeLegeHarUttaltSegUtoverTrygdemedisin: Boolean,
+    val rettsregelenErBenyttetFeil: Boolean,
+    val sakensDokumenter: Boolean,
+    val spraaketErIkkeTydelig: Boolean,
+    val utredningenAvAndreAktuelleForholdISaken: Boolean,
+    val utredningenAvArbeid: Boolean,
+    val utredningenAvEoesProblematikk: Boolean,
+    val utredningenAvInntektsforhold: Boolean,
+    val utredningenAvMedisinskeForhold: Boolean,
+    val veiledningFraNav: Boolean,
+    val vurderingAvFaktumErMangelfull: Boolean,
+    val arbeidsrettetBrukeroppfoelgingText: String?,
+    val betydeligAvvikText: String?,
+    val brukIOpplaeringText: String?,
+    val utredningenAvAndreAktuelleForholdISakenText: String?,
+    val utredningenAvArbeidText: String?,
+    val utredningenAvEoesProblematikkText: String?,
+    val utredningenAvInntektsforholdText: String?,
+    val utredningenAvMedisinskeForholdText: String?,
+    val veiledningFraNavText: String?,
+    val klageforberedelsenRadioValg: String?,
+    val utredningenRadioValg: String?,
+    val vedtaketRadioValg: String?,
+    val brukAvRaadgivendeLegeRadioValg: String?,
+) {
+    data class Date(
+        val weekNumber: Int,
+        val year: Int,
+        val month: Int,
+        val day: Int,
+        val iso: String,
+        val unix: Long
```

Comment:
Mener du med Unix antall ms siden 1970? Gir det mening for en LocalDate? Jeg er ikke sikker på om det gir mening for en LocalDateTime engang, må man ikke ha en tidssone for at det skal fungere?